gcp/resource_netapp_gcp_volume.go

```diff
@@ -469,8 +469,8 @@ func resourceGCPVolumeCreate(d *schema.ResourceData, meta interface{}) error {
 
 // Wait up to 15 minutes for volume creation to complete.
 func waitForVolumeCreationComplete(client *Client, volumeRes volumeResult) (volumeResult, error) {
-	waitSeconds := 900	// first volume creation can take 11 minutes
-	threshold := 900 -60	// when to warn
+	waitSeconds := 900    // first volume creation can take 11 minutes
```

Comment:
It would be nice to avoid space only changes.

Reply:
I agree. vscode runs "go fmt" automatically which I assume happened here.

Reply:
Agreed.  And we could run go fmt in our pipeline and error out if there is any change.